usage-collection/src/main/proto/usage.proto

```diff
@@ -1414,6 +1414,11 @@ message CDPProxyDetails {
   string authentication = 3;
 }
 
+message CDPOwnDnsZones {
+    // Whether DNS is provided for the postgres by the customer or not
+    bool postgres = 1;
+}
+
```

Comment:
Is it possible that in the future this message will have additional fields, to track some additional info regarding own dsn zones? If not, then a separate message is overkill for a single field and it should be simply a new bool field in CDPNetworkDetails

Reply:
Yes, there is some possibility for that, that's why I created a separate message for that.

Reply:
Also, on GCP there is also a similar feature of DNS zones, though it is currently not used by CDP.